src/tests/performance/proxy.conf

```diff
@@ -59,9 +57,7 @@ server default {
 
 	recv Accounting-Request {
 		if (!&Event-Timestamp) {
```

Comment:
this could likely be just

`&Event-Timestamp = "%l"  # only sets it if there's no Event-Timestamp`